spec/requests/public_recipes_spec.rb

```diff
@@ -0,0 +1,10 @@
+# require 'rails_helper'
+
+# RSpec.describe 'PublicRecipes', type: :request do
+#   describe 'GET /index' do
+#     it 'returns http success' do
+#       get '/public_recipes'
+#       expect(response).to have_http_status(:success)
+#     end
+#   end
+# end
```

Comment:
- [ ] Let's add tests in this file or remove them if we don't need them. We should not leave commented code because it makes our codebase look unclean.